package.json

```diff
@@ -18,7 +18,7 @@
   "devDependencies": {
```

Comment:
Please do not add useless files to your PR.
You can use **git add src/** command to add needed files
**git status** command — for checking file status